src/web-socket-handler.ts

```diff
@@ -74,6 +74,57 @@ export class WebSocketHandler implements WebSocketInterface {
         return true;
     }
 
+    public static restartableHandleStandardInput(
+        createWS: (() => Promise<WebSocket>),
+        stdin: stream.Readable | any,
+        streamNum: number = 0,
+        retryCount: number = 3,
+    ): (() => WebSocket | null) {
```

Comment:
Why does this return a function to return a WebSocket instead of a `Promise`?  I think this should probably just be an `async` function.